Make documentation language more inclusive (#2876)
microsoft/AzureTRE: docs/azure-tre-overview/airlock.md

The Airlock allows a TRE user to start the `import` or `export` process to a given workspace. A number of milestones must be reached in order to complete a successful import or export. These milestones are defined using the following states:

1. **Draft**: An Airlock request has been created but has not yet started. The TRE User/Researcher has now access to a storage location and they must identify the data to be processed. At this point the airlock import/export processes allow a single file to be processed. However a compressed file may be used (zip).

> 2. **Submitted**: The request was submitted by the ressearcher (not yet processed).

Reviewer: ```suggestion
2. **Submitted**: The request was submitted by the researcher (not yet processed).
```

## Approval mechanism

> The approval mechanism, is bundled with any airlock process, providing a specific way to `approve` or `reject` the data. This mechanism will allow the Airlock Managers to explicitly approve/reject the process, after having acess to the data. The Airlock Manager users will be able to execute a manual review on the data using the tools available to them in a reviewal TRE Workspace.

Reviewer: ```suggestion
The approval mechanism, is bundled with any airlock process, providing a specific way to `approve` or `reject` the data. This mechanism will allow the Airlock Managers to explicitly approve/reject the process, after having access to the data. The Airlock Manager users will be able to execute a manual review on the data using the tools available to them in a review TRE Workspace.
```

> Once this manual review is executed, Airlock Managers can proactivelly approve or reject the airlock request.

Reviewer: ```suggestion
Once this manual review is executed, Airlock Managers can proactively approve or reject the airlock request.
```
